frontend/src/app/firewood/forests/firewood-guidelines/forest-guidelines/how-to-cut/how-to-cut.component.html

```diff
@@ -119,47 +119,64 @@ <h3>What to expect</h3>
-<h3>Tagging your load</h3>
-<p>Before hauling wood out of the forest, you will need to fill out a load tag and attach it
-  to the back of your vehicle. The load tag helps law enforcement see that you have a valid
-  permit and have collected your wood legally.
-</p>
-<p>You'll get your load tags after you purchase your permit. Each load tag is equal to 1/2 cord of wood.</p>
-<h3>Filling out tags for online permits</h3>
-<p>Online permit buyers are emailed black and white load tags that you will need to print with your permit</p>
-<p>To validate your tag, you will need to:</p>
-<ul>
-  <li>Fill in the day and the month that you are harvesting the wood with a black marker. For example,
-    if you harvested on the 4th of July, you would fill in "4" for day and "7" for month.</li>
-  <li>Fold your tag on the dotted lines</li>
-  <li>Place each tag in a zip-top bag, and close the bag.</li>
-  <li>Nail your tag(s) to a piece of wood in the back of your load so that it is visible to cars behind you.</li>
-</ul>
-<h3>Filling out tags for in-person permits</h3>
-<p>You will be given colored load tags when you buy your permit from a district office or a registered vendor.</p>
-<p>To validate your tag, you will need to:</p>
-<ul>
-  <li>Punch out the month</li>
-  <li>Punch out the date</li>
-  <li>Nail your tag(s) to a piece of wood in the back of your load so that it is visible to cars behind you.</li>
-</ul>
+<!-- Tagging Your Load Block for Regions 1, 8, and 9-->
+<div *ngIf="(forest.forestAbbr) == 'ipnf' || (forest.forestAbbr)=='flathead'|| (forest.forestAbbr)=='conf' || (forest.forestAbbr)=='gwj' || (forest.forestAbbr)=='hiawatha'">
+  <h3>Load tags are required for permits bought online</h3>
+  <p>
+    If you get your permit online, you’ll need to tag your load by filling out
+    and attaching a load tag to your firewood before you haul it out of the forest.
+    You'll get your load tags after you purchase your permit.
+  </p>
+  <h3>How to tag your load</h3>
+  <p>After you purchase your permit online, you’ll be emailed black and white load tags along with your permit. You will need to print your load tags and your permit and bring them with you when you go to the forest.</p>
+  <p>Fill out your tag(s) after you've loaded your wood, but before you start hauling it.</p>
+  <h3>To fill out a tag, you'll need to:</h3>
+  <ul>
+    <li>Write in the day and the month that you get the wood with a black marker. For example, if you get wood  on the 4th of July, you would fill in "4" for day and "7" for month.</li>
+  </ul>
+  <h3>To attach a tag, you'll need to:</h3>
+  <ul>
+    <li>Fold your tag on the dotted lines.</li>
+    <li>Place each tag in a zip-top bag, and close the bag.</li>
+    <li>Nail your tag(s) to a piece of wood in the back of your load so that it is visible to cars behind you.</li>
+  </ul>
+</div>
+
+<!-- Tagging Your Load Block for Region 2 -->
+<div *ngIf="(forest.forestAbbr)=='shoshone' || (forest.forestAbbr)=='GMUG'">
```

Comment:
I'd prefer `(forest.forestAbbr) ===  'shoshone'` but no biggie, the code works.

Reply:
@briandavidson - I went ahead and corrected all equality operators to `===`